### DISCUSSION TITLE
Konica Minolta Bizhub 363 Error

### DISCUSSION THREAD

**Discussion by oswaldoguzman2022** [cyanfish/naps2#96] [Support]
Hello.

Sorry, my english is not good.

The last version not working controller twain Konica minolta Bizhub 363.

Teh device driver is:

https://dl.konicaminolta.eu/es?tx_kmdownloadcentersite_downloadproxy[fileId]=ca068be7dad3c53689d67b5f5dc3c3cf&tx_kmdownloadcentersite_downloadproxy[documentId]=135981&tx_kmdownloadcentersite_downloadproxy[system]=KonicaMinolta&tx_kmdownloadcentersite_downloadproxy[language]=ES&type=1558521685

In naps2 version 5.8.2 the twain driver works fine.

Thanks.

**Comment by cyanfish** [comment]
Hi, I recommend trying in your profile settings, select Advanced -> Twain Implementation -> Old DSM. That may help.

You also may want to try with the latest [beta version](https://www.naps2.com/download) and see if that helps.
